aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -982,6 +983,133 @@ void int8_gemm(
 }
 #endif // !defined(USE_ROCM) && !defined(_MSC_VER)
 
+
+#ifdef USE_ROCM
+
+template <>
+void trsm<float>(HIPBLAS_TRSM_ARGTYPES(float)) {
+  TORCH_HIPBLAS_CHECK(cublasStrsm(
+      handle, side, uplo, trans, diag, m, n, alpha, A, lda, B, ldb));
+}
+
+template <>
+void trsm<double>(HIPBLAS_TRSM_ARGTYPES(double)) {
+  TORCH_HIPBLAS_CHECK(cublasDtrsm(
+      handle, side, uplo, trans, diag, m, n, alpha, A, lda, B, ldb));
+}
```

Comment:
This looks like a verbatim copy of https://github.com/pytorch/pytorch/blob/854fe470cdc78873268bf3b8c8d392173f2aac8e/aten/src/ATen/cuda/CUDABlas.cpp#L985-L995

which makes me wonder, why hipifier can not take care of that one?